internal/sessiontest/session_test.go

```diff
@@ -28,7 +29,15 @@ func TestSigningSession(t *testing.T) {
 func TestDisclosureSession(t *testing.T) {
 	id := irma.NewAttributeTypeIdentifier("irma-demo.RU.studentCard.studentID")
 	request := getDisclosureRequest(id)
-	sessionHelper(t, request, "verification", nil)
+	responseString := sessionHelper(t, request, "verification", nil)
+
+	// Check default JWT validity
+	claims := jwt.MapClaims{}
+	jwt.ParseWithClaims(responseString, claims, nil)
+
+	iat := claims["iat"].(float64)
+	exp := claims["exp"].(float64)
+	require.True(t, iat+120000 == exp)
```

Comment:
Why the factor 1000 (i.e. not 120)? The `iat` and `exp` fields of JWTs are always in seconds according to the spec, and the `ResultJwtValidity` field of the `RequestorBaseRequest` is also in seconds according to its documentation.

Reply:
Totally stupid brainfart. Corrected to original expiration time calculation.